scripts/jwst/jwst.py

```diff
@@ -361,20 +288,18 @@ def _generate_examples(self, files, object_ids=None):
                         "image": [
                             {
                                 "band": data["image_band"][i][j].decode("utf-8"),
-                                "flux": data["image_array"][i][j],
-                                "mask": np.logical_or(
-                                    data["image_array"][i][j] == 0,
-                                    ~np.isfinite(data["image_array"][i][j])
-                                ).astype(bool),
+                                "flux": data["image_flux"][i][j],
+                                "ivar": data["image_ivar"][i][j],
+                                "mask": data["image_mask"][i][j].astype(bool),
                                 "psf_fwhm": data["image_psf_fwhm"][i][j],
                                 "scale": data["image_scale"][i][j],
                             }
-                            for j, _ in enumerate(self.config.bands)
+                            for j in range(len(data["image_band"][i]))
```

Comment:
This should probably throw a warning if the band in the data is not the same as the bands expected by the loader.